core-build/gulp-core-build-sass/src/CSSModules.ts

```diff
@@ -24,6 +24,7 @@ export interface ICSSModules {
 export default class CSSModules implements ICSSModules {
   private _classMap: IClassMap;
   private _rootPath: string;
+  private _generateScopedName: (name: string, fileName: string, css: string) => string;
```

Comment:
```suggestion
  private _generateScopedName: ((name: string, fileName: string, css: string) => string) | undefined;
```

Reply:
It's a little awkward that this name is the same (save for the leading `_`) between the private and public members.

Reply:
Good point, changed to `_customizedGenerateScopedName`